lib/christmas_tree_formatter.rb

```diff
@@ -1,40 +1,40 @@
 # encoding: UTF-8
 
-require "rspec/core"
-require "rspec/core/formatters/base_text_formatter"
+if defined? RSpec
```

Comment:
Yeah I think either way is fine. I mostly wanted to keep the `.rspec` file the same so it wouldn't have to change in projects using the formatter.